### PR TITLE
docs: add explanation into CHANGELOG

### PR DESCRIPTION
**Pull request by Flouse**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +11,10 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ### Features
 * Unlock legacy withdrawal cells feature ([#200](https://github.com/godwokenrises/light-godwoken/pull/200))
-* L1-transfer feature ([#210](https://github.com/godwokenrises/light-godwoken/pull/210))
+* Support layer1-transfer ([#210](https://github.com/godwokenrises/light-godwoken/pull/210))
+> This feature provides a handy function for users to send CKB to their CEX or wallet or other L1 addresses on Godwoken Bridge. It helps users transfer excess CKBs into sUDT units (usually withdrawal units), a tool that was missing in Nervos ecosystem.
```
Comment (Flouse) [inline CHANGELOG.md:15]:
```suggestion
> This feature provides a handy function for users to send CKB to their CEX or wallet or other L1 addresses through Godwoken Bridge. It helps users transfer excess CKBs into sUDT units (usually withdrawal units), a tool that was missing in Nervos ecosystem.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,7 +11,10 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ### Features
 * Unlock legacy withdrawal cells feature ([#200](https://github.com/godwokenrises/light-godwoken/pull/200))
```
Comment (Flouse) [inline CHANGELOG.md:13]:
```suggestion
* Support unlock legacy withdrawal cells from Godwoken_v0 ([#200](https://github.com/godwokenrises/light-godwoken/pull/200))
> This feature provides a convenient function for users to unlock legacy(v0) withdrawal cells through https://bridge.godwoken.io/#/v0/withdrawal/pending
> [What is legacy withdrawal cells (v0)?](https://github.com/godwokenrises/godwoken/blob/develop/docs/deposit_and_withdrawal.md#legacy-withdrawal-cells-v0)

```

**Comment by ShookLyngs** [comment]
Should we also change these:
- packages/light-godwoken/CHANGELOG.md
- apps/godwoken-bridge/CHANGELOG.md

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,8 +10,14 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
...
 
 ### Features
-* Unlock legacy withdrawal cells feature ([#200](https://github.com/godwokenrises/light-godwoken/pull/200))
-* L1-transfer feature ([#210](https://github.com/godwokenrises/light-godwoken/pull/210))
+* Support unlocking legacy withdrawal cells from Godwoken_v0 ([#200](https://github.com/godwokenrises/light-godwoken/pull/200))
+> This feature provides a convenient function for users to unlock legacy(v0) withdrawal cells through https://bridge.godwoken.io/#/v0/withdrawal/pending
+> [What is legacy withdrawal cells (v0)?](https://github.com/godwokenrises/godwoken/blob/develop/docs/deposit_and_withdrawal.md#legacy-withdrawal-cells-v0)
+
+* Support layer1-transfer ([#210](https://github.com/godwokenrises/light-godwoken/pull/210))
+> This feature provides a handy function for users to send CKB to their CEX or wallet or other L1 addresses through Godwoken Bridge. It helps users transfer excess CKBs into sUDT units (usually withdrawal units), a tool that was missing in Nervos ecosystem.
```
Comment (Flouse) [inline CHANGELOG.md:18]:
```suggestion
> This feature provides a handy function for users to send CKB to their CEX or wallet or other L1 addresses through Godwoken Bridge. It helps users to transfer the extra CKB inside a sUDT cell (usually a withdrawal cell), a tool that was missing in  Nervos ecosystem.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,8 +10,14 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
...
 ### Features
-* Unlock legacy withdrawal cells feature ([#200](https://github.com/godwokenrises/light-godwoken/pull/200))
-* L1-transfer feature ([#210](https://github.com/godwokenrises/light-godwoken/pull/210))
+* Support unlocking legacy withdrawal cells from Godwoken_v0 ([#200](https://github.com/godwokenrises/light-godwoken/pull/200))
+> This feature provides a convenient function for users to unlock legacy(v0) withdrawal cells through https://bridge.godwoken.io/#/v0/withdrawal/pending
+> [What is legacy withdrawal cells (v0)?](https://github.com/godwokenrises/godwoken/blob/develop/docs/deposit_and_withdrawal.md#legacy-withdrawal-cells-v0)
+
+* Support layer1-transfer ([#210](https://github.com/godwokenrises/light-godwoken/pull/210))
+> This feature provides a handy function for users to send CKB to their CEX or wallet or other L1 addresses through Godwoken Bridge. It helps users to transfer the extra CKB inside a sUDT cell (usually a withdrawal cell), a tool that was missing in  Nervos ecosystem.
+> Thank @alejandroRbit for [the feature request](https://github.com/godwokenrises/light-godwoken/issues/189).
```
Comment (ShookLyngs) [inline CHANGELOG.md:19]:
```suggestion
> Thanks [@alejandroRbit](https://github.com/alejandroRbit) for [the feature request](https://github.com/godwokenrises/light-godwoken/issues/189).
```

Do you think we should add a link to the `@`? 
Will GitHub add it automatically if we don't?

**Review by ShookLyngs — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -10,8 +10,14 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
...
 ### Features
-* Unlock legacy withdrawal cells feature ([#200](https://github.com/godwokenrises/light-godwoken/pull/200))
-* L1-transfer feature ([#210](https://github.com/godwokenrises/light-godwoken/pull/210))
+* Support unlocking legacy withdrawal cells from Godwoken_v0 ([#200](https://github.com/godwokenrises/light-godwoken/pull/200))
+> This feature provides a convenient function for users to unlock legacy(v0) withdrawal cells through https://bridge.godwoken.io/#/v0/withdrawal/pending
+> [What is legacy withdrawal cells (v0)?](https://github.com/godwokenrises/godwoken/blob/develop/docs/deposit_and_withdrawal.md#legacy-withdrawal-cells-v0)
+
+* Support layer1-transfer ([#210](https://github.com/godwokenrises/light-godwoken/pull/210))
+> This feature provides a handy function for users to send CKB to their CEX or wallet or other L1 addresses through Godwoken Bridge. It helps users to transfer the extra CKB inside a sUDT cell (usually a withdrawal cell), a tool that was missing in  Nervos ecosystem.
+> Thanks [@alejandroRbit](https://github.com/alejandroRbit) for [the feature request](https://github.com/godwokenrises/light-godwoken/issues/189).
```
Comment (Flouse) [inline CHANGELOG.md:19]:
```suggestion
> Thanks @alejandroRbit for [the feature request](https://github.com/godwokenrises/light-godwoken/issues/189).
```

**Review by ShookLyngs — APPROVED** [review]

**Pull request merged**